Modules/vc-module-content/VirtoCommerce.ContentModule.Data/Converters/MenuLinkConverter.cs

```diff
@@ -0,0 +1,66 @@
+using System;
```

Comment:
Move all this code to the relevant data entities methods, as we have done for other modules